### DISCUSSION TITLE
Error: file was opened locally from a previous session. To restore it, go to track settings and reopen the file

### DISCUSSION THREAD

**Discussion by MehdiiYac** [GMOD/jbrowse-components#3455] [Q&A] [answered]
Hello,
I loaded some data into Jbrowse (version 2.3.3), after refreshing the page i keep running into the following error:

Error: file ("iwgsc_refseqv2.1_assembly.fa") was opened locally from a previous session. To restore it, go to track settings and reopen the file

Do you know a was to fix this issue or how to open the track settings,

Thank you

**Comment by cmdcolin** [comment, marked as answer]
Hi there MehdiiYac, generally I would recommend loading the genome using the command line e.g. 
```
npm install -g @jbrowse/cli
samtools faidx iwgsc_refseqv2.1_assembly.fa #create the fai file (fasta index)
jbrowse add-assembly iwgsc_refseqv2.1_assembly.fa
```

It is true with that error that when you open up a file from your local harddrive in the web browser, it won't be visible after page refresh. Web browsers don't make it easy to do this access files from your harddisk persistently for security reasons (though in the future, there may be features for this...)

Hope that helps!
